packages/dialect-react-ui/components/NotificationsButton/index.tsx

```diff
@@ -98,7 +97,7 @@ function WrappedNotificationsButton(props: PropTypes): JSX.Element {
           // style={{ backdropFilter: 'blur(132px)' }}
         >
           <Notifications
-            channels={props.channels}
+            channels={props.channels || ['web3', 'email', 'sms', 'telegram']}
```

Comment:
maybe extract it out of component? so that we eliminate possible rerendering issue